var/spack/repos/builtin/packages/libmesh/package.py

```diff
@@ -19,29 +19,182 @@ class Libmesh(Package):
     version('1.2.1', sha256='11c22c7d96874a17de6b8c74caa45d6745d40bf3610e88b2bd28fd3381f5ba70')
     version('1.0.0', 'cb464fc63ea0b71b1e69fa3f5d4f93a4')
 
+    # support for libraries that are only available through the bundled copies:
+    # TODO libMesh 1.2.1 gained the ability to specify a path to capnproto
+    variant('capnproto', default=False, description='Compile with the bundled capnproto serialization library')
+    variant('exodusii', default=False, description='Compile with the bundled ExodusII output library')
+    variant('fparser', default=False, description='Compile with the bundled fparser function parser library')
+    variant('gmv', default=False, description='Compile with the bundled gmv format IO library')
+    variant('laspack', default=False, description='Compile with the bundled laspack interative solver library')
+    variant('libhilbert', default=False, description='Compile with the bundled libHilbert partitioning library')
+    variant('metaphysicl', default=False, description='Compile with the bundled metaphysicl AD library')
+    variant('metis', default=False, description='Compile with the bundled METIS graph partitioning library')
+    variant('nanoflann', default=False, description='Compile with the bundled nanoflann graph library')
+    variant('nemesis', default=False, description='Compile with the bundled nemesis IO library')
+    variant('netcdf', default=False, description='Compile with the bundled NETCDF IO library')
+    # TODO parmetis assumes that we use the bundled copy of metis, but the
```

Comment:
That’s not the case for Spack, it’s parametris is patched to build against external Metis

Reply:
That comment (its vague; I will go fix it) refers to the copy of parmetis bundled with libMesh itself. More exactly:
1. libMesh can only use the copy of parmetis that it bundles with itself
2. libMesh can use either the bundled metis or metis from PETSc
3. libMesh's parmetis copy must use the bundled copy of metis

Hence libMesh's bundled parmetis conflicts with an external metis installation.